addons/binding/org.openhab.binding.homie/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,\
```

Comment:
Yes, I'll post a link here for future google search reference :)
https://www.eclipse.org/forums/index.php/t/1088038/

I managed to achieve my tasks, Thanks!